packages/foundations/scss/icons/include.scss

```
@@ -1,6 +1,13 @@
 @use "icon-helpers";
 @use "../default.assets-paths" as *;
 
+%icon-alignment {
+	&:is(span) {
+		display: inline-flex;
+		vertical-align: baseline;
+	}
+}
+
```

```suggestion
[data-icon],
[data-icon-before],
[data-icon-after] {
	&:is(span) {
		display: inline-flex;
		vertical-align: baseline;
	}
}

```

You could instead write it like this and than you wouldn't need to `SCSS placeholder` and references within the other selectors.